specs/lib/parseGLTFSpec.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+var parseGLTF = require('../../lib/parseGLTF');
+
+describe('parseGLTF', function() {
```

Comment:
Also add a test for when the shader is a data uri, e.g., like this:

https://github.com/KhronosGroup/glTF/blob/master/sampleModels/box/glTF-Embedded/box.gltf#L202-L210

If `fs.readFile` does not support data uris (it might not given its name), see if there is a built-in way to handle them in node or a library on npm; otherwise, check out how we do it client-side: https://github.com/AnalyticalGraphicsInc/gltf-utilities